container/go/cmd/pusher/pusher.go

```diff
@@ -58,7 +59,7 @@ func main() {
 	}
 
 	// Ensure src is a directory, trim basename index.json if not.
-	if filepath.Base(*src) == "index.json" {
+	if filepath.Base(*src) == "index.json" || filepath.Base(*src) == "manifest.json" {
```

Comment:
I think this logic could be improved. Right now it will only work if the path given is to a index.json or manifest.json. Instead, I would recommend using [this](https://golang.org/pkg/os/#FileMode.IsDir) function to check if the given path is a directory and if not, trim the path. Also generate logs here because it's changing what the input is being interpreted as.